py3status/module.py

```diff
@@ -325,7 +325,7 @@ def set_module_options(self, module):
 
             if 'align' in mod_config:
                 align = mod_config['align']
-                if not (isinstance(align, basestring) and
+                if not isinstance(align, basestring) and (
```

Comment:
why have you done this?
explain your working and why is this not in the PR description

Reply:
Travis.

Reply:
I don't see why it is part of this PR maybe I've missed something 
and please don't resolve issue I raise I'm not happy

Also I don't have to help you with your code I can just NAK it

Reply:
`../module.py|328 col 25 warning| line break after binary operator [W504]`

We have this in `tox.ini` to skip those. 
```ini
[pytest]
flake8-ignore =
# line break after binary operator (2018-Aug-22)
    W504

# invalid escape sequence (2018-Aug-22)
    W605
```

When we `black`, we can remove `W504` later so Travis will complain next time. `W605` is complaining about several `\?if=`. I'm just seeing this warning here and it didn't take much to remove the warning. I'll undo this.

Reply:
but wtf is this being done in this PR for - how many times do I have to say the same shit?

Reply:
Because that's how I fixed errors/warnings in first place. I forget I had this blacklisted.